Configuration/Generator/python/QQH1252T_14TeV_TuneCP5_cfi.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CP5Settings_cfi import *
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         # put here the efficiency of your filter (1. if no filter)
+                         filterEfficiency = cms.untracked.double(1.0),
+                         comEnergy = cms.double(14000.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         # put here the cross section of your process (in pb)
+                         crossSection = cms.untracked.double(0.388),
+                         maxEventsToPrint = cms.untracked.int32(3),
```

Comment:
Suggest to set to 0 for relval.

Reply:
Do you mean cross section or which parameter to set to zero?